src/app/tests/suites/TV_AccountLoginCluster.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Account Login Cluster Tests
+
+config:
+    cluster: "Account Login"
+    endpoint: 3
+
+tests:
+    - label: "Get Setup PIN Command"
+      command: "GetSetupPIN"
+      arguments:
+          values:
+              - name: "tempAccountIdentifier"
+                value: "asdf"
```

Comment:
Should there be a `response:` section here that expects some specific response back?  Something like:
```
      response:
          values:
              - name: "returnValue"
                value: "somethingsomething"
```
depending on what the return value is supposed to be?

Reply:
Updated.

Reply:
Added: 
`response:
          values:
              - name: "setupPIN"
                value: "tempPin123"`